stix_shifter_modules/guardium/stix_translation/json/qsearch_from_stix_map.json

```diff
@@ -0,0 +1,46 @@
+{
+
+  "x-ibm-finding": {
+      "fields": {
+        "finding_type": ["finding_type"],
+        "start": ["startTime"],
+        "end": ["endTime"],
+        "database_name": ["Database"],
+        "db_type":["db_type"],
+        "Severity":["Severity"]
```

Comment:
The to-stix map is mapping severity to `x-guardium` so you would also want it mapped the same for the from-six mappings. Also make sure the STIX property name is all lowercase.

Reply:
done